distribution/openrct2.d.ts

```diff
@@ -4948,4 +4951,11 @@ declare global {
         getAllObjects(type: "scenery_group"): SceneryGroupObject[];
         getAllObjects(type: "music"): LoadedObject[];
     }
+
+    /**
+     * Interface to handle the plugin manager
+     */
+    interface PluginManager {
+        readonly plugins: PluginMetadata[];
```

Comment:
Where is the PluginMetadata type defined? I think if we merge this PR we should not have this missing.

Reply:
It existed already, isn't it this:

https://github.com/OpenRCT2/OpenRCT2/blob/f10911194507a88ca3e05d8f1f35b738d27ae353/distribution/openrct2.d.ts#L140-L153

Reply:
Didn't realize its already there, nevermind then.